src/main/java/no/unit/nva/cristin/projects/model/nva/NvaContributor.java

```diff
@@ -0,0 +1,44 @@
+package no.unit.nva.cristin.projects.model.nva;
+
+import static no.unit.nva.cristin.projects.JsonPropertyNames.AFFILIATION;
+import static no.unit.nva.cristin.projects.JsonPropertyNames.IDENTITY;
+import static no.unit.nva.cristin.projects.JsonPropertyNames.TYPE;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import nva.commons.core.JacocoGenerated;
+
+//import java.time.Instant;
+
+@JacocoGenerated
+@JsonPropertyOrder({TYPE, IDENTITY, AFFILIATION})
+public class NvaContributor {
```

Comment:
I need some feedback about naming model classes. Should we prefix with Nva... and Cristin... or something else?

Reply:
Nva.. and Cristing.. are good for now I suppose

Reply:
I'd call the internal stuff just Contributor, while I'd call the external stuff CristinX. I'd also consider modules to separate the models